airflow/providers/amazon/aws/sensors/glue.py

```diff
@@ -0,0 +1,65 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from airflow.exceptions import AirflowException
+from airflow.providers.amazon.aws.hooks.glue import AwsGlueJobHook
+from airflow.sensors.base_sensor_operator import BaseSensorOperator
+from airflow.utils.decorators import apply_defaults
+
+
+class AwsGlueJobSensor(BaseSensorOperator):
+    """
+    Waits for an AWS Glue Job to reach any of the status below
+    'FAILED', 'STOPPED', 'SUCCEEDED'
+
+    :param job_name: The AWS Glue Job unique name
+    :type job_name: str
+    :param run_id: The AWS Glue current running job identifier
+    :type run_id: str
+    """
+    template_fields = ('job_name', 'run_id')
+    JOB_POLL_INTERVAL = 6
```

Comment:
```suggestion
```
What is this line for? You already have this in the hook where you are using it.